db/controllers/comments.controller.js

```diff
@@ -11,4 +11,13 @@ function postCommentWithId(req, res, next) {
     .catch(next)
 }
 
-module.exports = { postCommentWithId }
\ No newline at end of file
+function deleteComment(req, res, next) {
+    const id = req.params.comment_id
+    deleteCommentById(id).then((comment) => {
+        res.status(200).send({"deleted_comment" :comment})
```

Comment:
204 and nothing to send - but you will still need to invoke send to receive the status code in your test